lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -2079,14 +2084,15 @@ deleteTransaction ctx (ApiT wid) (ApiTxId (ApiT (tid))) = do
 
 listTransactions
     :: forall ctx s k n. (ctx ~ ApiLayer s k)
-    => ctx
-    -> ApiT WalletId
-    -> Maybe MinWithdrawal
-    -> Maybe Iso8601Time
-    -> Maybe Iso8601Time
-    -> Maybe (ApiT SortOrder)
+    => ctx -- ^
+    -> ApiT WalletId -- ^
+    -> Maybe MinWithdrawal -- ^
+    -> Maybe Iso8601Time -- ^
+    -> Maybe Iso8601Time -- ^
+    -> Maybe (ApiT SortOrder) -- ^
+    -> Bool -- ^ metadata json schema
```

Comment:
If I'm not mistaken the new query parameter is not documented in the swagger, so should be added there.